tools/gce/linux_performance_worker_init.sh

```diff
@@ -126,6 +126,7 @@ sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotne
 sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
 sudo apt-get update
 sudo apt-get install -y dotnet-dev-1.0.0-preview2-003131
```

Comment:
no, because it's still needed for older releases - at least v1.2.x (it can be eventually removed, but I'll much rather do that in a few months).